src/settings/Token.js

```diff
@@ -38,9 +84,23 @@ class Token extends React.Component {
                 id="token"
                 name="token"
                 label={<FormattedMessage id="ui-developer.setToken.authenticationToken" />}
+                onChange={this.handleTokenChange}
```

Comment:
I am not 100% convinced it's right to update this as soon as the field changes but delay the actual update of the token until the Save button is pressed. Don't you think both those things should happen at the same time?

Reply:
This allows you to inspect the token before you save it. So, no, I don't think this should be tied to clicking "Save".

Reply:
OK.